__init__.py

```diff
@@ -221,8 +303,23 @@ def apply_user_vars(self, uvars):
 
         self.define_param('year', str(date.today().year))
 
+        # Select alternatives to use for protocol analyzers.
+        if not self.lookup_param("unit_resp") or self.lookup_param("unit_resp") == self.lookup_param("unit_orig"):
```

Comment:
Good catch. I can't get an empty responder to pass through if a require `unit_resp`. Changing it so that the user now always has to provide both, and then we check if they are equal.